.gitlab-ci.yml

```diff
@@ -36,14 +39,17 @@ generator:
   image: registry.ddbuild.io/images/mirror/golang:alpine
   tags: ["arch:amd64"]
   cache: *go-cache
-  script:
-    - apk add --no-cache gomplate
-    - gomplate --config .gitlab/config.yaml
   artifacts:
     paths:
       - .gitlab/pipeline-bottlecap.yaml
       - .gitlab/pipeline-go-agent.yaml
       - .gitlab/pipeline-lambda-extension.yaml
+    reports:
+      dotenv: .env
+  script:
+    - if [[ "$CI_COMMIT_TAG" =~ ^v[0-9]+$ ]]; then echo "VERSION=$(echo "${CI_COMMIT_TAG##*v}" | cut -d. -f2)" >> .env; fi
```

Comment:
Allows us to send a parsed version of the layer for the `publish images` job which requires it as an environment variable. This is the only way to send it before those jobs get generated.

Reply:
should the env file `export ENV`?

Reply:
I'm not sure I understand the question